projects/fundamental-ngx/src/lib/action-bar/action-bar-back.component.spec.ts

```diff
@@ -0,0 +1,30 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { ActionBarBackComponent } from './action-bar-back.component';
+
+describe('ActionBarBackComponent', () => {
+    let component: ActionBarBackComponent;
+    let fixture: ComponentFixture<ActionBarBackComponent>;
+
+    beforeEach(async(() => {
+        TestBed.configureTestingModule({
+            declarations: [ActionBarBackComponent]
+        }).compileComponents();
+    }));
+
+    beforeEach(() => {
+        fixture = TestBed.createComponent(ActionBarBackComponent);
+        component = fixture.componentInstance;
+        fixture.detectChanges();
+
+        spyOn(component, '_setProperties').and.callThrough();
+        spyOn(component, '_addClassToElement');
+    });
+
+    it('should create', () => {
+        expect(component).toBeTruthy();
+        component.ngOnInit();
+        expect(component._setProperties).toHaveBeenCalled();
+        expect(component._addClassToElement).toHaveBeenCalledWith('fd-action-bar__back');
```

Comment:
i see those kind of tests on each component - maybe we can think about of abstracting it and testing it once. Otherwise is the same code in each component